front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleModal.tsx

```diff
@@ -74,6 +74,8 @@ const ImportTrainScheduleModal = ({
   const { t } = useTranslation(['operationalStudies/importTrainSchedule']);
   const rollingStockID = useSelector(getRollingStockID);
 
+  const { refetch: refetchTimetable } =
+    osrdEditoastApi.endpoints.getTimetableById.useQuerySubscription({ id: timetableId });
```

Comment:
It's a great use of useQuerySubscription but if you add the `timetable` tag in `invalidatesTags` of the `postTrainScheduleStandaloneSimulation` endpoint, you don't need that endpoint call at all.

Reply:
See my comment above :top: